test/e2e/network_test.go

```diff
@@ -368,21 +368,24 @@ func getAffinityForNode(node *v1.Node) (*v1.Affinity, error) {
 	}, nil
 }
 
-// ensureNamespace checks if a namespace with the provided name exists and creates one if it does not
-func (tc *testContext) ensureNamespace(name string) error {
+// ensureNamespace checks if a namespace with the provided name exists and creates one if it does not with the given
+// labels
+func (tc *testContext) ensureNamespace(name string, labels map[string]string) error {
 	// Check if the namespace exists
 	_, err := tc.client.K8s.CoreV1().Namespaces().Get(context.TODO(), name, metav1.GetOptions{})
 	if err != nil && !apierrors.IsNotFound(err) {
 		return err
 	}
 	if err == nil {
+		// at this point; trust the namespace was properly created with the required labels
```

Comment:
You can assert that the labels are present by looking at the object returned which is currently ignored.

Reply:
Not sure if is worth spending an API call on such assertion, given the probability is very low or [none](https://github.com/openshift/windows-machine-config-operator/pull/1230#discussion_r960774958).